values.yaml

```diff
@@ -275,8 +275,8 @@ proxy:
 # If expose the service via "ingress", the Nginx will not be used
 nginx:
   image:
-    repository: goharbor/nginx-photon
-    tag: v1.10.17
+    repository: yangjiao0817/nginx-photon
```

Comment:
is it  correct to use prefix yangjiao0817?

Reply:
that's for testing purpose. It has been updated now. Thanks